packages/visual-editor/src/ui/utils/oauth-based-opal-shell.ts

```diff
@@ -203,37 +207,30 @@ export class OAuthBasedOpalShell implements OpalShellHostProtocol {
   }
 
   /**
-   * Use this method to tweak the body of the request if necessary.
+   * Attaches the access token to the body of the request.
    */
-  #maybeAugmentInit(url: string, init: RequestInit, accessToken: string) {
+  #attachAccessToken(init: RequestInit, accessToken: string) {
```

Comment:
Suggest `#injectAccessTokenIntoJsonBody` and matching name for the config property